content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md

```diff
@@ -26,7 +26,7 @@ sub-project](https://github.com/kubernetes/community/blob/master/sig-architectur
 bit about yourself, your role and how you got involved in Kubernetes.
 
 **Riaan Kleinhans (RK)**: Hi! My name is Riaan Kleinhans and I live in
-South Africa. I am the Project manager for the [ii-Team](ii.nz) in New
+South Africa. I am the Project manager for the [ii-Team](https://ii.nz) in New
```

Comment:
Though trivial, We can accept this change as a good practice

Reply:
I see. Do I need to revert the changes to the old content?

Reply:
Yes, I'd do that

Reply:
done